server/nbd-svc-routines.c

```diff
@@ -1277,7 +1294,21 @@ int nbd_handle_request(int sock, int threads)
     nbd_update_json_config_file(dev);
     pthread_mutex_unlock(&dev->lock);
 
+err1:
```

Comment:
we can remove the err1 label and move the g_thread_pool_free under err label, like:

```
err: 
if (thread_pool)
    g_thread_pool_free(thread_pool, false, true);
```

Reply:
Agree.